src/pages/TestDetailPage/TestDetailPage.tsx

```diff
@@ -0,0 +1,209 @@
+  const loading = typeof loadingFromProps === 'boolean' ? loadingFromProps : loadingFromHook;
+  const error = typeof errorFromProps === 'string' ? errorFromProps : errorFromHook;
+
+  const handleStart = () => {
+    if (!test) return;
+    if (onStart) {
+      onStart(test.id);
+      return;
+    }
+    window.location.href = `/tests/${test.id}/run`;
+  };
+
+  return (
+    <Root>
+      {/* Left navigation area — visual mock showing where NavigationSidebar goes. Keep as a placeholder but styled to match mockup */}
+      {/* Visual recuadro where NavigationSidebar should be integrated */}
+      <NavigationBox data-testid="navigation-box">NavigationSidebar</NavigationBox>
+      {/* Separator line placed outside the navigation box */}
+      <Separator data-testid="navigation-separator" />
+
+      <MainArea>
+        <PageHeader>
+          <div>
+            <HeaderTitle>Tests For You !</HeaderTitle>
+            <HeaderSubtitle>Here is your Profile Dashboard</HeaderSubtitle>
+          </div>
+
+          <HeaderRight>
+            <SearchBox>
+              <SearchBar data-testid="searchbar-component" placeholder="Search" size="small" />
+            </SearchBox>
+            <AvatarWrapper>
+              <AvatarMenu data-testid="avatar-component" avatarUrl="" username="" />
+            </AvatarWrapper>
+          </HeaderRight>
+        </PageHeader>
+
+        <InnerCard>
+          {loading ? (
+            <Box display="flex" alignItems="center" justifyContent="center" height={200}>
+              <CircularProgress aria-label="loading" />
+            </Box>
+          ) : error ? (
+            <PlaceholderBox role="alert">{`Error loading test: ${error}`}</PlaceholderBox>
+          ) : test ? (
+            <>
+              <HeaderRow>
+                <div style={{ flex: 1, display: 'flex', alignItems: 'center', gap: 24 }}>
+                  <LogoArea>
+                    {/* Logo is imported from local assets so colors are preserved. */}
+                    {/* File to add: src/pages/TestDetailPage/assets/tcs-inline-logo.jpg */}
+                    {/* Once you upload the JPG I will create it; until then this import is a placeholder. */}
+                    {/* Use Vite-friendly URL import so Storybook/Vite can resolve assets */}
+                    <LogoImage
+                      src={new URL('./assets/logo.jpg', import.meta.url).href}
+                      size="medium"
+                      alt="TCS logo"
+                      data-testid="inline-logo"
+                    />
+                  </LogoArea>
+
+                  <TitleSection>
+                    <TitleText
+                      sx={{
+                        width: 'auto',
+                        whiteSpace: 'normal',
+                        overflow: 'visible',
+                        textOverflow: 'unset',
+                      }}
+                    >
+                      TCS Quiz Competition
```

Comment:
Title and subtitle in the card are hardcoded. These are user-visible and should be localized or passed from data/props.

**Sugerencia:**
```typescript
Use a translation function or pass title/subtitle via props (and default to test.name/test.company) instead of hardcoding strings.
```